atlasdb-workload-server-distribution/src/main/java/com/palantir/atlasdb/workload/WorkloadServerLauncher.java

```diff
@@ -119,12 +121,11 @@ private void scheduleBackgroundJobs(Environment environment) {
                         AntithesisCassandraSidecarResource.INSTANCE,
                         DefaultBuggifyFactory.INSTANCE),
                 0,
-                10,
+                2,
                 TimeUnit.SECONDS);
     }
 
     private void runWorkflows(WorkloadServerConfiguration configuration, Environment environment) {
-        // This is a single threaded executor; this is intentional, so that we only run one workflow at a time.
```

Comment:
As far as I can tell this is still true, so we should probably leave the comment (if we don't want to "only run one workflow at a time" we can reword it to say we want to test with running workflows in parallel or something)

Reply:
yikes -- yeah I tried running multiple workflows in parallel, and that broke things :( will re-add the comment